warsawlo/schema.py

```diff
@@ -116,6 +117,54 @@ class Meta:
         interfaces = (relay.Node,)
 
 
+class CharInFilter(django_filters.BaseInFilter, django_filters.CharFilter):
+    pass
+
+
+class SearchFilter(django_filters.FilterSet):
+    QUERY_FIELD = "school_name"
+    SCHOOL_TYPE_GENERALISED = "szkoła ponadpodstawowa"
+
+    class Meta:
+        model = School
+        fields = ["school_type", "is_public"]
+
+    query = django_filters.CharFilter(method="query_filter")
+
+    district = django_filters.CharFilter(field_name="address__district")
+
+    # TODO: Filter input as list not string
+    # or filtering
+    available_extended_subjects = CharInFilter(
```

Comment:
This should be probably MultipleChoiceFilter?

Reply:
Done